crates/api_common/src/utils.rs

```diff
@@ -140,19 +140,6 @@ pub fn is_top_mod(
   }
 }
 
-/// Marks a post as read for a given person.
-#[tracing::instrument(skip_all)]
-pub async fn mark_post_as_read(
-  person_id: PersonId,
-  post_id: PostId,
-  pool: &mut DbPool<'_>,
-) -> LemmyResult<()> {
-  PostRead::mark_as_read(pool, HashSet::from([post_id]), person_id)
-    .await
-    .with_lemmy_type(LemmyErrorType::CouldntMarkPostAsRead)?;
-  Ok(())
```

Comment:
Most of these changes are boilerplate, because I've gotten rid of this pointless wrapper function, by having `PostRead::mark_as_read` return the correct LemmyError.